README.md

```diff
@@ -5,11 +5,19 @@
 Birdcage is a cross-platform embeddable sandboxing library allowing restrictions
 to Filesystem and Network operations using native operating system APIs.
 
-Birdcage **is not** a complete sandbox preventing all side-effects or permanent
-damage. Applications can still execute most system calls, which is especially
-dangerous when execution is performed as root. Do not use Birdcage as a safety
-barrier for known-malicious code and keep other security mechanisms like user
-restrictions in place.
+Birdcage was originally developed for use the [Phylum CLI] as an extra layer of
+protection against potentially malicious dependencies (see the [blog post] for
+details). To better protect yourself from these security risks, [sign up now]!
+
+[phylum cli]: https://github.com/phylum-dev/cli
+[blog post]: https://blog.phylum.io/sandboxing-package-installations-arms-developers-with-defense-against-open-source-attacks-and-unintended-consequences/
+[sign up now]: https://www.phylum.io/
+
+Birdcage focuses **only** on Filesystem and Network operations. It **is not** a
+complete sandbox preventing all side-effects or permanent damage. Applications
+can still execute most system calls, which is especially dangerous when
+execution is performed as root. Birdcage should be combined with other security
+mechanisms, especially if you are executing known-malicious code.
 
 ## Usage
```

Comment:
I've already changed the examples in my stab at it. I'd say it is an improvement but feel free to tweak or discard:

## Example

An example for using Birdcage's API can be found in `./examples/sandbox`, which
runs an application with CLI-configurable restrictions applied.

Trying to run without an exceptions will produce an error:

```bash
$ cargo run --example sandbox -- echo "Hello, Sandbox\!"
Error: Os { code: 13, kind: PermissionDenied, message: "Permission denied" }
```

Running the same command with explicit permissions allows execution:

```bash
$ cargo run --example sandbox -- -e /usr/bin/echo -e /usr/lib echo "Hello, Sandbox\!"
Hello, Sandbox!
```

Check out `cargo run --example sandbox -- --help` for more information on how to
use the example.
```